libraries/alicloud_backend.rb

```diff
@@ -173,6 +173,7 @@ def initialize(opts)
   def validate_parameters(allow: [], required: nil, require_any_of: nil)
     if required
       raise ArgumentError, "Expected required parameters as Array of Symbols, got #{required}" unless required.is_a?(Array) && required.all? { |r| r.is_a?(Symbol) }
+      raise ArgumentError, "#{@__resource_name__}: region must be provided via environment variable or hash parameter" if required.include?(:region) && (!@opts.is_a?(Hash) || (@opts[:region].nil? || @opts[:region] == ""))
```

Comment:
```suggestion
      raise ArgumentError, "#{@__resource_name__}: region must be provided via environment variable or hash parameter" if (@opts[:region].nil? || @opts[:region].empty?)
```

Reply:
also this can go below your Hash check [here](https://github.com/inspec/inspec-alicloud/pull/53/files#diff-533ce34f00798db00b4356b57c535ac15a505d19d572cf4be6703594c43145f2R177)